src/main/java/gregtech/api/nuclear/fission/components/CoolantChannel.java

```diff
@@ -0,0 +1,82 @@
+package gregtech.api.nuclear.fission.components;
+
+import gregtech.api.capability.ICoolantHandler;
+import gregtech.api.unification.material.Material;
+import gregtech.api.unification.material.properties.PropertyKey;
+
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoolantChannel extends ReactorComponent {
+
+    private final Material coolant;
+    private double weight;
+    private final List<Pair<FuelRod, FuelRod>> fuelRodPairs = new ObjectArrayList<>();
+
+    private ICoolantHandler inputHandler;
+    private ICoolantHandler outputHandler;
+    public double partialCoolant;
```

Comment:
is this used anywhere(or set?)

Reply:
It is used in the fission reactor, in the case that the coolant is really good at absorbing heat and thus would output less than a full liter of coolant per second.